### ISSUE TITLE
Is the current page object accessible in a shortcode?

### ISSUE THREAD

**Issue by bert-bruggeman** [11ty/eleventy#566]
Is it possible to use the current page object in shortcodes or the config file? In jekyll I used below method to add modal content or footnotes to the end of the page for shortcodes like a button or image with lightbox... 

**for example:**

I could set a variable "footerContent" with an empty array for all pages
`footerContent: []`

In each shortcode I would add an item to "footerContent"
```
page.footerContent.push(item)
return '<button class="modal-btn">...</button>'
```

In the page layout I can then loop through the array...
```

</body>
{% for item in footerContent%}
{{item}}
{% endfor%}
```

**Comment by supaht** [comment]
I'm also interested in this. 

I have a shortcode for inserting lazy-loading images in blog markdown files. But I'm currently working on setting up an RSS feed and the lazy loading images are not working in the feed. So it would be great if I could detect in the shortcode function if I'm in the `feed.njk` page and return non-lazy-loaded images.

**Comment by zachleat** [comment]
@bert-bruggeman are you asking about a specific template language?

**Comment by zachleat** [comment]
This shipped in 0.11.0. https://github.com/11ty/eleventy/releases/tag/v0.11.0 See #741
